Pipfile.lock

```diff
@@ -1,412 +0,0 @@
-{
```

Comment:
You're erroneously deleting this file -- we don't want to do that!

You should be able to get it back with:
```
git reset HEAD~1
git restore Pipfile.lock
git add .
git commit --amend
git push --force-with-lease
```